src/Modules/CalcOffence.lua

```diff
@@ -3919,4 +3936,36 @@ function calcs.offence(env, actor, activeSkill)
 		output.CullingDPS = output.CombinedDPS * (output.CullMultiplier - 1)
 	end
 	output.CombinedDPS = output.CombinedDPS * output.CullMultiplier
+
+	if activeSkill.mirage and activeSkill.mirage.output and activeSkill.mirage.output.TotalDPS then
+		local mirageCount = activeSkill.mirage.count or 1
+		output.MirageDPS = activeSkill.mirage.output.TotalDPS * mirageCount
+		output.CombinedDPS = output.CombinedDPS + activeSkill.mirage.output.TotalDPS * mirageCount
+
+		if activeSkill.mirage.output.IgniteDPS and activeSkill.mirage.output.IgniteDPS > output.IgniteDPS then
+			output.MirageDPS = output.MirageDPS + activeSkill.mirage.output.IgniteDPS
```

Comment:
This logic seems like it might be misleading for some users.  Is it possible to set `output.IgniteDPS` to the mirages' ignite and show the source?  That might be more appropriate in `skillDPS`, though, iirc

Reply:
I will add to `skillDPS` source `igniteSource = activeSkill.activeEffect.grantedEffect.name + " (Mirage)"` (same for bleed). I doubt it will ever happen as mirages all have a hefty "less damage" on them, but just in case the code is there.